bin/misc-functions.sh

```diff
@@ -223,6 +223,12 @@ install_qa_check() {
 				echo "${obj} ${needed}"	>> "${PORTAGE_BUILDDIR}"/build-info/NEEDED
 				echo "${arch#EM_};${obj};${soname};${rpath};${needed}" >> "${PORTAGE_BUILDDIR}"/build-info/NEEDED.ELF.2
 			done <<< ${scanelf_output}
+
+			# scanelf's output ordering is non-deterministic.
+			local needed
+			for needed in "${PORTAGE_BUILDDIR}"/build-info/NEEDED{,.ELF.2}; do
+				sort -o "${needed}" "${needed}"
```

Comment:
Output will vary according to the current locale.
```suggestion
				LC_ALL=C sort -o "${needed}" "${needed}"
```

Reply:
Thanks for the suggestion. Applied.